3542/1/lib/repo.rb

```diff
@@ -0,0 +1,63 @@
+require 'httparty'
+require 'nokogiri'
+require 'open-uri'
+
+#:reek:TooManyInstanceVariables
+class Repo
+  URI = 'https://api.github.com/search/repositories'.freeze
+
+  attr_reader :name, :used_by, :watched_by, :stars, :forks, :contributors, :issues
+
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Security/Open
+  def initialize(gem_name)
+    @api_response = HTTParty.get(URI, query: { q: gem_name })
+    @repo = repository
+    @html = Nokogiri::HTML(open(@repo[:html_url]))
+    @name = @repo[:name]
+    @stars = @repo[:watchers_count]
+    @forks = @repo[:forks_count]
+    @issues = @repo[:open_issues_count]
+    @contributors = contributors_count
+    @watched_by = watched_by_count
```

Comment:
For this and below same as
https://github.com/rubizza-camp/summer-2019/pull/303/files#diff-da83bc4a88208591f850ddf5e21733d4R21